src/components/Header/Header.js

```diff
@@ -191,6 +191,16 @@ const Header = ({location}) => {
     },
   });
 
+  // Check if stored current organization belongs to user profile
```

Comment:
Line 182 is still going to cause issues when a user is logged in but doesn't have any organizations yet bc organizations.edges[0] is undefined. I'm trying to think how we want to refactor this logic .. 🤔

Reply:
I think you need something like this starting around line 175: 
```javascript
  // Check if an org is cached
  var currentOrg;
  try {
    currentOrg = JSON.parse(localStorage.getItem('currentOrganization'));
  } catch (e) {
    currentOrg = null;
  }
  // Check if stored current organization belongs to user profile
  const organizations = profile && profile.organizations;
  const hasOrgs = organizations && organizations.edges.length > 0;
  const allowedOrgs = hasOrgs
    ? organizations.edges.map(({node}) => node.id)
    : [];
  if (!currentOrg || !allowedOrgs.includes(currentOrg.id)) {
    currentOrg = hasOrgs && organizations.edges[0].node;
    console.log('Setting org ' + currentOrg);
    localStorage.setItem('currentOrganization', JSON.stringify(currentOrg));
  }
  const {refetch} = useQuery(ALL_STUDIES, {
    fetchPolicy: 'network-only',
    variables: {
      organization: currentOrg && currentOrg.id,
    },
  });

```
It's basically what you had before but removes that old block of code